package.json

```diff
@@ -29,6 +29,7 @@
     "jsonfile": "^2.4.0",
     "node-forge": "^0.7.1",
     "recursive-readdir": "^2.1.1",
+    "salesforce-alm": "^40.0.6",
```

Comment:
It is useful to be tied to its own version so it can upgrade when it wants to without breaking.